features_hq/support/step_helper.rb

```diff
@@ -0,0 +1,35 @@
+module Helpers
+  
+  DEFAULT_ADMIN = {
+    :email => 'default@man.net',
+    :password => 'defaultpass'    
+  }
+
+  def create_admin_user email= DEFAULT_ADMIN[:email], password= DEFAULT_ADMIN[:password]
+    AdminUser.new(email: email, password: password, password_confirmation: password).save!
+  end
+
+  def login email= DEFAULT_ADMIN[:email], password= DEFAULT_ADMIN[:password]
+    visit new_admin_user_session_path
+    fill_in "admin_user_email", :with => email
+    fill_in "admin_user_password", :with => password
+    click_button "Login"
+  end
+
+  def logout
+    visit destroy_admin_user_session_path
```

Comment:
I would think in an integration test `session.clear` would not be the way to go, because an end-user cannot accomplish that directly